### PR TITLE
Laravel 6 - Logging changes introduced in Laravel 5.6

### PR DESCRIPTION
**Pull request by matthewnessworthy**
For Laravel 5.6 (and above) some changes were made to logging. Specifically, the creation of a new Monolog driver is now more cleanly handled via a dedicated process.
A separate logging installation is now provided for those on 5.6 and above.

### CODE REVIEW

**Comment by iamacarpet** [comment]
Hello @matthewnessworthy ,

I'm happy to accept the dev prefix change, as long as it doesn't change how existing projects work if they don't configure anything (which it looks like you've handled, thank you).

I don't think we'll be able to accept the stuff for Laravel 5.6+ in this branch though, as we are specifically targeting 5.5 LTS in the main branch. Are you able to split this into two PRs?

Our next supported release will be Laravel 6.0 LTS, although for us this is a way off at the moment - I am happy to create a dev branch to merge this into, if that is the version you are trying to target?

Regards,
iamacarpet

**Comment by matthewnessworthy** [comment]
Hi @iamacarpet,

All these changes so far are 5.5+ compatible. 
I don't mind if you want to setup a Laravel 6 branch, whatever is easy for you.
So far there is very little that is incompatible with Laravel 6, so the upgrade should be fairly easy.

File: README.md [hunk 1]
```diff
@@ -99,6 +99,14 @@ $app = new A1comms\GaeSupportLaravel\Foundation\Application(
 A1comms\GaeSupportLaravel\Log\Logger::setup($app);
 ```
 
+*NOTE:* If you are using Laravel 5.6 or above then logging can be setup via a custom logging channel in `config/logging.php`:
```
Comment (iamacarpet) [inline README.md]: Are you ok to remove the reference to L5.6 here please?

Our intention is to only support LTS releases, so we'd prefer not to confuse things.

If this is the recommended method for L6.0, please drop our recommendation to use `Logger::setup` above and replace it with your method.

File: src/config/gaesupport.php [hunk 2]
```diff
@@ -18,4 +18,6 @@
     'trace_providers' => [
 
     ],
-];
\ No newline at end of file
+
+    'dev-prefix' => 'dev-',
```
Comment (iamacarpet) [inline src/config/gaesupport.php]: Are you ok to split the dev prefix stuff into another commit please?

I'd like to be able to cherry-pick it back into the L5.5 branch to remain as consistent as possible.

**Review by iamacarpet — CHANGES_REQUESTED** [review]
Thanks @matthewnessworthy ,

Your effort is greatly appreciated.

I'll be very happy to merge this, but can I request a few changes first please?

**Review by iamacarpet — APPROVED** [review]
Thanks for the changes, looks great!
